src/main/java/com/ihren/exercise4/models/ElementMapper.java

```diff
@@ -0,0 +1,11 @@
+package com.ihren.exercise4.models;
+
+import lombok.experimental.UtilityClass;
+import java.util.Optional;
+
+@UtilityClass
```

Comment:
Mapper can not be util class. Even here map method has static and it is not ok but it was initial state. So, you should remove @UtilityClass